students/Snayxts/module-2-html/exercises/zdk6/zvuk.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Monthly Top 5 - Free Music Archive</title>
+</head>
+<body>
+    <h1>Free Music Archive</h1>
+    <br>
+
+    <h2>Monthly Top 5 (December 2019)</h2>
+    <h3>#1</h3>

```

Comment:
Osobno bih ovdje pristupio s <ol> elementom jer bih time naglasio da je popis pjesama dan u strogo određenom redoslijedu. Time bih riješio problem numeriranja pjesama, a naslov bih potencijalno dodao na osnovu imena izvođača i imena pjesme. Trenutno, naslov "#1" ne znači puno sam po sebi kada se dokument primjerice isprinta. Zanima nas što je točno "#1", zar ne? :)